issue_07/script.js

```diff
@@ -0,0 +1,50 @@
+const div = document.getElementById('js-loading');
+const loadingImg = document.createElement('img');
+loadingImg.src = 'loading-circle.gif';
+loadingImg.alt = 'ローディング画像';
```

Comment:
Let's put this part into a function instead of a variable.

Reply:
I have corrected the areas you pointed out.
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/c6e32585fd52ade77912c65ef0f33d1fd28dee0d